expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java

```diff
@@ -197,6 +198,21 @@ void shouldReturnFramedIndexDocumentFromResource(Class<?> instanceType)
         assertThat(indexDoc.fetchId(), is(not(nullValue())));
     }
 
+    @Test
+    void shouldReturnIndexDocumentWithContextUri()
+        throws JsonProcessingException, NotFoundException {
+
+        Publication publication = randomPublication(AcademicArticle.class)
+                                      .copy()
+                                      .withEntityDescription(new EntityDescription())
+                                      .build();
+
+        Resource resourceUpdate = Resource.fromPublication(publication);
+        ExpandedResource indexDoc = (ExpandedResource) expansionService.expandEntry(resourceUpdate);
+        assertThat(indexDoc.getAllFields().get("@context"),
+                   is(equalTo("https://api.dev.nva.aws.unit.no/publication/context")));
```

Comment:
Read api host from environment?

Reply:
I'm not keen on reading from env in tests…we don't test the expectation, just the code. If someone mutates the string, it won't fail